biomodels/src/test/java/org/reactome/release/ModelsTSVParserTester.java

```diff
@@ -2,68 +2,95 @@
     }
 
     @Test
-    public void testModels2PathwaysCorrectlyParsed() {
-        Map<String, List<String>> pathwayToBioModelsIds =
-            ModelsTSVParser.parse("src/test/resources/models2pathways.tsv");
-        assertEquals(pathwayToBioModelsIds.keySet().size(), correctNumberPathwaysInTestFile);
+    public void testFileReturnsPopulatedMap() {
+        String filepath = Paths.get(RESOURCE_DIR, "models2pathways.tsv").toFile().getAbsolutePath();
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse(filepath);
+        assertThat(pathwayToBioModelsIds.keySet().size(), is(equalTo(correctNumberPathwaysInTestFile)));
+    }
 
+    @Test
+    public void testFileReturnsCorrectNumberOfIds() {
+        String filepath = Paths.get(RESOURCE_DIR, "models2pathways.tsv").toFile().getAbsolutePath();
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse(filepath);
         long bioModelsIdsCount =
-            pathwayToBioModelsIds.keySet().stream().map(
-                pathway -> pathwayToBioModelsIds.get(pathway)
-            ).flatMap(
-                Collection::stream
-            ).distinct().count();
-        assertEquals(bioModelsIdsCount, correctNumberBioModelsIdsInTestFile); // Number of distinct BioModels Ids in test input file
+                pathwayToBioModelsIds.keySet().stream().map(
+                        pathway -> pathwayToBioModelsIds.get(pathway)
+                ).flatMap(
+                        Collection::stream
+                ).distinct().count();
+        assertThat(bioModelsIdsCount, is(equalTo(correctNumberBioModelsIdsInTestFile))); // Number of distinct BioModels Ids in test input file
+    }
 
+    @Test
+    public void testFileContainsCorrectMapping() {
+        String filepath = Paths.get(RESOURCE_DIR, "models2pathways.tsv").toFile().getAbsolutePath();
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse(filepath);
         // Tests pathway contains correct BioModels id
-        assertTrue(pathwayToBioModelsIds.get("R-HSA-199418").contains("BIOMD0000000175"));
+        assertThat(pathwayToBioModelsIds.get("R-HSA-199418").contains("BIOMD0000000175"), is(equalTo(true)));
+    }
 
+    @Test
+    public void testFileProducesMappingWithoutNullKeysOrValues() {
+        String filepath = Paths.get(RESOURCE_DIR, "models2pathways.tsv").toFile().getAbsolutePath();
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse(filepath);
         // Tests for null in Map keys and value
-        assertTrue("pathwayToBioModelsIds map does not contain any null keys", !pathwayToBioModelsIds.containsKey(null));
-        assertTrue("pathwayToBioModelsIds map does not contain any null values", !pathwayToBioModelsIds.containsValue(null));
+        assertThat("pathwayToBioModelsIds map does not contain any null keys", pathwayToBioModelsIds.containsKey(null), is(equalTo(false)));
```

Comment:
This can (theoretically -- I haven't tested :) be:

`assertThat("pathwayToBioModelsIds map does not contain any null keys", pathwayToBioModelsIds, not(hasKey(nullValue())));`